en/frames_vtol/vtol_quadplane_foxtech_loong_2160.md

```diff
@@ -0,0 +1,369 @@
+
+### Sensors
+
+#### Pitot Tube
+
+1. The sensor can be installed with 2x M3x16mm screws in the front right corner of the baseplate.
+   Take care that the connector is facing the center of the fuselage.
+
+   ![Mounted airspeed sensor](../../assets/airframes/vtol/foxtech_loong_2160/11-airspeed-sensor.jpg)
+
+   Only the front tube (not as shown in the picture) is used; the other tube can be removed since our experience showed that the pressure inside the fuselage is sufficient as static pressure.
+
+1. When the stack is mounted inside the fuselage, the tube coming from the wing and the one from the airspeed sensor need to be spliced together.
+   Use some spit to push them together and afterward use a heat shrink tube to reinforce the connection.
+
+   :::warning
+   Use a heat source carefully since the foam starts to melt at high temperatures.
+   :::
+
+#### Lidar
+
+1. Mark the location to install the lidar with some tape or a pen.
+   Cut a hole inside the PVC shell and the foam, so that the lidar fits in place.
+
+   ![Prepared lidar hole](../../assets/airframes/vtol/foxtech_loong_2160/12-lidar-01.jpg)
+
+1. Secure the lidar with hot glue.
+
+   ![Installed lidar](../../assets/airframes/vtol/foxtech_loong_2160/13-lidar-02.jpg)
+
+#### GPS/Compass
+
+1. Use double sided tape to mount the GPS in the rear of the vehicle underneath the rear latch.
+
+   ![Installed GPS](../../assets/airframes//vtol/foxtech_loong_2160/14-gps.jpg)
+
+   The arrow on the GPS for the orientation can be ignored.
+   The orientation will be detected by the flight controller during the calibration.
+
+### Flight Controller
+
+Install either the Pixhawk or Skynode onto the baseplate.
+
+#### Pixhawk 6c/6c mini
+
+1. Use double sided tape to mount the flight controller to the baseplate.
+
+#### Skynode
+
+1. Use 4x M3x8 screws to mount the Skynode to the baseplate.
+   Make sure that the top of the "A" is facing to the front of the vehicle.
+1. Plug the 40A Power Module into the upper one of the two power connectors.
+1. Plug one (or if needed two) USB adapters into the 4-pin JST-GH connectors into the back of the Skynode and feed them to the front of the plate.
+   Fix the cables with zip ties in place.
+1. Tape a I2C splitter to the front right side of the baseplate (The splitter can be used to plug in ETH devices such as a radio link.)
+1. Connect the I2C splitter with the ETH port in the back of the Skynode.
+1. Plug in the two 40-pin cables into the front of the Skynode.
+1. Plug in the USB-C extension cable and bend it over to the front.
+   The bend needs to be very tight, so that the plate will fit into the vehicle.
+
+![Installed Skynode](../../assets/airframes/vtol/foxtech_loong_2160/15-skynode.jpg)
+
+#### Adapter Boards
+
+1. Screw the Pixhawk adapter Boards to the top plate.
+
+### Antennas and RC Receiver
+
+1. Tape the Skynode LTE antennas to the side of the fuselage as shown in the picture:

```

Comment:
FYI only, I've made each of the steps into lists rather than paragraphs, where possible. I find it very much easier to follow steps if they are written as steps. Also very much easier to map what I see with where I am at in the setup if the image is associated with a particular step.